static/app/types/event.tsx

```diff
@@ -381,11 +386,25 @@ type OSContext = {
   version: string;
 };
 
+export enum OtelContextKey {
+  ATTRIBUTES = 'attributes',
+  RESOURCE = 'resource',
+}
+
+// OpenTelemetry Context
+// https://develop.sentry.dev/sdk/performance/opentelemetry/#opentelemetry-context
+interface OtelContext extends Partial<Record<OtelContextKey, unknown>>, BaseContext {
+  type: 'otel';
+  [OtelContextKey.ATTRIBUTES]?: Record<string, unknown>;
+  [OtelContextKey.RESOURCE]?: Record<string, unknown>;
+}
+
 type EventContexts = {
   client_os?: OSContext;
   device?: DeviceContext;
   feedback?: Record<string, any>;
   os?: OSContext;
+  otel?: OtelContext;
```

Comment:
since this is a known context, shall we add it to the following list?

https://github.com/getsentry/sentry/blob/f3db0c04d9c035f52ba1fedcda235a13a9b57b7e/static/app/components/events/contextSummary/index.tsx#L19

Reply:
Great idea, will do!